### PR TITLE
fix: Remove personID from events query

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

After discussion of how to handle the lots-of-distinctids problem, we realised we could just omit the personID. 

Or maybe not...
## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts [hunk 1]
```diff
@@ -384,7 +384,6 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                                 ],
                                 orderBy: ['timestamp ASC'],
                                 limit: 1000000,
-                                personId: String(person.id),
```
Comment (pauldambra) [inline frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts:387]: I don't think we can...

We do two queries

One for events with the session id
And another for the person whose session it was, in the time range of the session, but without the session id.

If we remove the person id then in the second case we query for _all_ events for that team in that time range, no?

**Review by pauldambra — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts (hunk 1 above)
Comment (benjackwhite) [reply frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts:387]: Oh yeah just realised that...

**Comment by pauldambra** [comment]
I think we have two problems here:

1. what do we do for session recording that we can land today
2. how do we fix this for hogql

Maybe they are the same. But we can't have a situation where you can't query events unless each person has less than N distinct ids 🙈

**Comment by thmsobrmlr** [comment]
> I think we have two problems here:
> 
> 1. what do we do for session recording that we can land today
> 2. how do we fix this for hogql
> 
> Maybe they have the same solution. But we can't have a situation where you can't query events unless each person has less than N distinct ids 🙈

Re. 1: Would it be possible to use a specific distinct_id instead of the person_id?

**Pull request closed without merging**